content/en/edge-computing.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Edge Computing
+status: Feedback Appreciated
+category: Technology
+---
+
+## What it is
+
+Edge computing is a [distributed system](/distributed-systems/) approach that shifts some storage and computing capacity from the primary data center to the data source.
+The gathered data is computed locally (e.g., on a factory floor, in a store, or throughout a city) rather than sent to a centralized data center for processing and analysis. 
+These local processing units or devices represent the system's edge, whereas the data center is its center.
+The output computed at the edge is then sent back to the primary data center for further processing.
+Examples of edge computing include gadgets on our wrists or computers that analyze traffic flow.
+
+## Problem it addresses
+
+Due to the enormous amounts of data that Internet of Things (IOT)devices create, data has become the next important asset for enterprises worldwide. 
```

Comment:
```suggestion
Over the past decade, we've seen an increasing amount of smart devices.
In some cases, real-time data processing is not only a nice-to-have but vital. 
Think of self-driving cars. 
Now imagine the data from sensors would have to be transferred to a data center for processing to be sent back to the vehicle so it can take action. 
The inherent network latency could be fatal. 
While this is an extreme example, most users wouldn't want to use a smart device unable to provide instant feedback. 
Today's world of the Internet of Things (IoT) calls for a new paradigm: edge computing. 
```
I took some of your points but rewrote this section with an example to visualize it better.